paper-tab.html

```diff
@@ -169,8 +164,25 @@
 
           anchor.click();
         }
-      }
+      },
+
+      _createRipple: function() {
+        var ripple = Polymer.PaperInkyFocusBehaviorImpl._createRipple();
+        ripple.removeAttribute('center');
+        ripple.classList.remove('circle');
+        return ripple;
+      },
 
+      _asyncClick: function() {
```

Comment:
I vaguely remember commenting on this, but can't see where the comment went -- why is this needed? Can you at least add a comment why it's needed, because it doesn't seem to be called anywhere in this file.